### PR TITLE
Add triangles

### LINKED ISSUE #30: Add triangle or polygons
It would be nice to have traingles, or perhaps more generally polygons.

### PR DESCRIPTION
**Pull request by albinahlback**
Added triangle, which can be filled.

Resolves #30

### CODE REVIEW

File: src/utility.jl [hunk 1]
```diff
@@ -0,0 +1,27 @@
+function area(s::Triangle)
```
Comment (aviks) [inline src/utility.jl:1]: I don't like having files called `utils` or `utility`, they quickly attract too much bad code. I'd suggest adding these methods to screen.jl (and maybe, later renaming _screen.jl_ to _geom.jl_ or some such)

File: src/utility.jl [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  abs(
+      s.p1[1] * (s.p2[2] - s.p3[2]) +
+      s.p2[1] * (s.p3[2] - s.p1[2]) +
+      s.p3[1] * (s.p1[2] - s.p2[2])
+     ) / 2
+end
```
Comment (aviks) [inline src/utility.jl:7]: this function should probably call `area_triangle`?

File: src/utility.jl [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+function inside(s::Triangle, p::Vector{Int})
+  A1 = area_triangle(p, s.p2, s.p3)
+  A2 = area_triangle(s.p1, p, s.p3)
+  A3 = area_triangle(s.p1, s.p2, p)
+  return area(s) == A1 + A2 + A3
+end
+
+function norm(u::Vector{Number}, p=2)
+  p != 2 && error("p = ", p, "not implemented yet.")
```
Comment (aviks) [inline src/utility.jl:25]: "Norm with p = 3 not implmented yet"

**Comment by albinahlback** [comment]
As you said, @aviks, (the presented format of) the `utility.jl` is unnecessary. I removed it completely.

**Comment by aviks** [comment]
Thanks. Sorry it took a long time for me to get back to this.

**Pull request merged**